src/tasks/python/PythonExtensionHelper.ts

```diff
@@ -0,0 +1,116 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// This will eventually be replaced by an API in the Python extension. See https://github.com/microsoft/vscode-python/issues/7282
+
+import * as fse from 'fs-extra';
+import * as os from 'os';
+import * as path from 'path';
+import * as vscode from "vscode";
+import CliDockerClient from '../../debugging/coreclr/CliDockerClient';
+import { delay } from '../../utils/delay';
+import { PythonDefaultDebugPort, PythonFileTarget, PythonModuleTarget } from '../../utils/pythonUtils';
+
+export namespace PythonExtensionHelper {
+    export interface DebugLaunchOptions {
+        host?: string;
+        port?: number;
+        wait?: boolean;
+    }
+
+    export function getDebuggerEnvironmentVars(): { [key: string]: string } {
+        return { ["PTVSD_LOG_DIR"]: `/dbglogs` };
```

Comment:
Nit: I think this should read `return { 'PTVSD_LOG_DIR': '/dbglogs' };`. In any case, can use single quotes, but I don't think `PTVSD_LOG_DIR` needs to be boxed.